web/libs/datamanager/src/components/MainView/GridView/GridView.scss

```diff
@@ -0,0 +1,76 @@
+
+.grid-view {
+  --column-count: 4;
+
+  &__resize {
+    width: 100%;
+  }
+
+  &__list {
+    padding: 0 0 16px;
+  }
+
+  &__cell {
+    padding: 16px 16px 0 0;
+    box-sizing: border-box;
+    
+    @mixin nthChild($columnCount) {
+      &:nth-child(4n) {
+        padding-right: 0;
+      }
+    }
+    
+    @include nthChild(4n);
+    // &:nth-child(4) {
+    //   padding-right: 0;
+    // }
```

Comment:
Flagging this as it is technically going to output the correct behaviour but the usage of a dynamic css var for the nth-child expression is not supported, hence this output below. Maybe we should just clean this up as it won't ever be supported, and if we want differing grid column strategies this should be done against its own ticket to cleanup the GridView.
```
    @mixin nthChild($columnCount) {
      &:nth-child(4n) {
        padding-right: 0;
      }
    }
    
    @include nthChild(4n);
    // &:nth-child(4) {
    //   padding-right: 0;
    // }
```

Reply:
opted to go in another direction but i needed to tweak how we do the grid so its class dependant